Automattic/wp-calypso
# composite-checkout: Clean up public api

#### Changes proposed in this Pull Request

This applies some fixes and improvements to the public API of `composite-checkout`. See each individual commit for the details.

#### Testing instructions
- Add a file called `packages/composite-checkout/demo/private.js` and inside it put the following: `export const stripeKey = 'TEST KEY HERE'`;
- Run `npm run composite-checkout-demo`.
- Verify that the form works as expected
